### PR TITLE
[TH2-368] Add README for sim-template.

### PR DESCRIPTION
**Pull request by Ilya-Loginov-Exactpro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Overview
+Simulator is a passive th2 component with parameterised rules which is implemented simulate logic.
+Script or other components can create, remove and get information about running rules in simulator via gRPC. Simulator can interact with conn (Connects).
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:3]: Let's replace information related to the sim (core) feature to link to the th2-sim project.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+This project implemented gRPC API described in the [th2-grpc-sim-template](https://github.com/th2-net/th2-grpc-sim-template/blob/master/src/main/proto/th2_grpc_sim_template/sim_template.proto "sim_template.proto")
+ 
+This project is template how create and add custom rules.
```
Comment (Nikita-Smirnov-Exactpro) [inline README.md:7]: This project demonstrates how to organize your project with custom rules.

**Review by Nikita-Smirnov-Exactpro — COMMENTED** [review]
Please add a chapter about what should we replace in this project when creating custom.
You can find the example in the th2-grpc-generator-template.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+This type has the most flexible check conditions. Rules of this type will be triggered if your custom logic in method checkTriggered return true.
+
+## Service
+If you want to add possibility for create a rule via gRPC you should edit [th2-grpc-sim-template](https://github.com/th2-net/th2-grpc-sim-template/blob/master/src/main/proto/th2_grpc_sim_template/sim_template.proto "sim_template.proto") and class [TemplateService](https://github.com/th2-net/th2-sim-template/blob/master/src/main/kotlin/com/exactpro/th2/sim/template/service/TemplateService.kt "TemplateService.kt").
+For add a rule to simulator you can use utility method ``ServiceUtils.addRule`` or method from ``Simulator`` class with name ``addRule``. On gRPC request you should return ``RuleID``.
+
+## Work example
+
+On picture present example of work simulator with enabled rule ``TemplateFixRule``. This rule send ``ExecutionReport`` message if income message is ``NewOrderSingle``.
+If rule in simulator income wrong message (not ``NewOrderSingle``), rule will not generate outgoing message.
```
Comment (jean-michel-perli-sonsin) [inline README.md:35]: **If the income message is wrong (not ``NewOrderSingle``), the rule in simulator** will not generate **an** outgoing message.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+
+## Service
+If you want to add possibility for create a rule via gRPC you should edit [th2-grpc-sim-template](https://github.com/th2-net/th2-grpc-sim-template/blob/master/src/main/proto/th2_grpc_sim_template/sim_template.proto "sim_template.proto") and class [TemplateService](https://github.com/th2-net/th2-sim-template/blob/master/src/main/kotlin/com/exactpro/th2/sim/template/service/TemplateService.kt "TemplateService.kt").
+For add a rule to simulator you can use utility method ``ServiceUtils.addRule`` or method from ``Simulator`` class with name ``addRule``. On gRPC request you should return ``RuleID``.
+
+## Work example
+
+On picture present example of work simulator with enabled rule ``TemplateFixRule``. This rule send ``ExecutionReport`` message if income message is ``NewOrderSingle``.
+If rule in simulator income wrong message (not ``NewOrderSingle``), rule will not generate outgoing message. 
+If rule in simulator income message is ``NewOrderSingle``, rule will generate one ``ExecutionReport``.
```
Comment (jean-michel-perli-sonsin) [inline README.md:36]: **If the income message is correct (NewOrderSingle), the** rule in simulator will generate one ``ExecutionReport``.

**Review by jean-michel-perli-sonsin — CHANGES_REQUESTED** [review]

**Pull request merged**